### PR TITLE
Sync metadata with problem-specifications

### PR DESCRIPTION
**Pull request by kytrinyx**
This syncs metadata for those exercises that do not have
pending changes to the tests.

This brings in updated URLs to ensure that they use
https rather than http.

### CODE REVIEW

File: exercises/practice/nth-prime/.meta/config.json
Comment (devkabiir) [inline exercises/practice/nth-prime/.meta/config.json:18]: `problem=8` does not seem to be relevant here, I think `problem=7` should be kept

**Review by devkabiir — CHANGES_REQUESTED** [review]

File: exercises/practice/nth-prime/.meta/config.json
Comment (kytrinyx) [reply exercises/practice/nth-prime/.meta/config.json:18]: That's a really good catch, thank you.

**Review by devkabiir — APPROVED** [review]

File: exercises/practice/nth-prime/.meta/config.json
Comment (kytrinyx) [reply exercises/practice/nth-prime/.meta/config.json:18]: I've submitted a PR upstream and pinged you as a reviewer. (We need 3 approvals to get PRs merged in the problem-specifications repository, so it might take a day to get to where I can fix this PR).

Comment (devkabiir) [reply exercises/practice/nth-prime/.meta/config.json:18]: I don't know if I have write access to that repository, so my approval might not count 😛

Comment (devkabiir) [reply exercises/practice/nth-prime/.meta/config.json:18]: > We need 3 approvals to get PRs merged in the problem-specifications repository

I know, I've been strategically avoiding making any PRs there if you haven't noticed already from our discussions.

Comment (kytrinyx) [reply exercises/practice/nth-prime/.meta/config.json:18]: All maintainers have write-access to the repository, if I am not mistaken.

> I've been strategically avoiding making any PRs there

Ah, now that you mention it, I can see that. And given the history of the repository, I can empathize!

**Review by Stargator — APPROVED** [review]

**Pull request merged**
